### PR TITLE
PICMI - Embedded boundary

### PR DESCRIPTION
**Pull request by roelof-groenewald**
Added functionality to picmi.py to allow including an embedded boundary from a python run script. Also added a test of that functionality.

### CODE REVIEW

File: Regression/prepare_file_travis.py
Comment (ax3l) [inline Regression/prepare_file_travis.py:132]: ok, but mid-term I would say we should try to make `USE_EB` on by default so we can cover it well.
(And only adding runtime overhead/data if EBs are used from inputs)

**Review by ax3l — APPROVED** [review]
LGTM, waiting for @dpgrote for final approval.

**Review by dpgrote — APPROVED** [review]
This looks good to me. Thanks!

**Pull request merged**
